### PR TITLE
[feature24] Another pull_request testing

### PR DESCRIPTION
**Pull request by nprott**
Great course at Udemy :)

### CODE REVIEW

File: feature24/new_test.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This file is a simple practice of the pull-request on the public repository.
```
Comment (muminek74) [inline feature24/new_test.txt:1]: **Great job, soon you will become a GitHub master!!!**

File: feature24/cool-new_testing.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Another new file to test the pull request
```
Comment (muminek74) [inline feature24/cool-new_testing.txt:1]: Looks like the testing is going well

File: feature24/new_test.txt (hunk 1 above)
Comment (nprott) [reply feature24/new_test.txt:1]: Thanks for the approval :-)

File: feature24/cool-new_testing.txt (hunk 2 above)
Comment (nprott) [reply feature24/cool-new_testing.txt:1]: It seems so...I'm looking forward to some more complicated stuff!

**Pull request merged**